src/foam/nanos/logger/StdoutLoggerDAO.js

```diff
@@ -30,8 +36,16 @@ foam.CLASS({
       LogMessage lm = (LogMessage) getDelegate().put_(x, obj);
       if ( getEnabled() &&
            lm != null ) {
-        System.out.println(lm.getCreated() + ","+lm.getThread()+","+lm.getSeverity()+","+lm.getMessage());
-      } 
+        // Only write WARN, ERROR to SYSLOG in production to reduce
+        // burden on syslogd, journald. With our own journal logs we are
+        // effectively writing out logs twice. 
+       AppConfig appConfig = (AppConfig) x.get("appConfig");
```

Comment:
Not the end of the world, but couldn't we look up the appConfig just once?

Reply:
changed.